hnn_core/pyramidal.py

```diff
@@ -49,30 +73,36 @@ def __init__(self, gid, soma_props):
         self.sect_loc = dict()
         # for legacy use with L5Pyr
         self.list_dend = []
```

Comment:
In the spirit of cleaning up cell class attributes, is `Pyr.list_dend` necessary given that we have the keys in `Pyr.dends`?

Reply:
I guess we can save this for another PR as you mentioned above....